app/views/publish/_autocomplete_warning.html.erb

```diff
@@ -1,8 +1,6 @@
-<% autocomplete_warning.messages.each do |message| %>
+<% autocomplete_warning.message.each do |message| %>
```

Comment:
is there only one message now per presenter? if so do we need to loop? if not we could keep both methods